test/domain/relation/establish.spec.ts

```diff
@@ -17,4 +19,18 @@ describe('domain/relation/establish', () =>
 
         expect(promise).rejects.toStrictEqual(new RelationAlreadyExists);
     });
+
+    it('Should rollback created data after failure', async () =>
+    {
+        const promise = establish(UNKNOWN_REQUESTER, CREATOR1);
```

Comment:
The rollback can only be tested when the creation of a relation succeeds. This is not the case here, meaning that non of the rollback functions are called.